less/availity-angular.less

```diff
@@ -10,3 +10,17 @@
   background: none;
   z-index: 1011;
 }
+
+.dimmer-content {
```

Comment:
Take a look at the changes I have made, the base dimmer-content is closer to what you said, then I added dimmer-content-cenetered and dimmer-content-dark to give more options